.travis.yml

```diff
@@ -2,5 +2,5 @@ language: python
 python: 3.7
 dist: xenial
 cache: pip
-before_install: pip install -U pre-commit
+before_install: pip install -r sep-requirements.txt
```

Comment:
What is "sep"?

Reply:
Separation.

Reply:
However, it is possible to name the cheks-requirements.txt or test-requirements.txt.

Reply:
It should be `test`, separation doesn't make sense. Are you going to call other files separation1, separation2, separation6?

Reply:
Ok, thanks.